src/main.rs

```diff
@@ -244,4 +240,20 @@ struct Entry {
     pub count: usize,
     pub workspace_packages: Vec<String>,
     pub version: String,
+    pub path: Option<PathBuf>,
+}
+
+impl Entry {
+    fn to_toml(&self) -> Item {
+        let version = Value::String(Formatted::new(self.version.clone()));
+
+        Item::Value(if let Some(path) = &self.path {
+            let mut itable = InlineTable::new();
```

Comment:
Definitely, I need support for those fields (instead of the current `version = "*"` that gets emitted but obviously wasn't going to conflate this PR with that :grimacing: